samples/highcharts/demo/venn-point-legend/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Venn diagram
+authors:
+  - Jon Arild Nygård
+js_wrap: b
+tags:
+  - Highcharts demo
+categories:
+  - More chart types
```

Comment:
Tags and categories should be removed, this applies only when part of the demos in the website.